### PR TITLE
Read all buffers from DUT console before the measurement starts

### PR DESCRIPTION
**Pull request by wtrgjadmw**
This PR adds an implementation that waits to read all buffers from the DUT serial console.
Wait 1 second and repeat until no data is readable.

### CODE REVIEW

File: docs/servo_power_test/src/dut_controller.ts [hunk 1]
```diff
@@ -28,6 +28,33 @@ export class DutController {
     const chunk = await this.dut.read();
     return chunk;
   }
+  private async checkDutBuffer() {
```
Comment (hikalium) [inline docs/servo_power_test/src/dut_controller.ts:31]: the name "check" is not friendly to developers to know what value will be returned in which case.
How about `readDataWithTimeout()` and add a function-level documentation that describes the meaning of the return value?

Comment (hikalium) [reply docs/servo_power_test/src/dut_controller.ts:31]: Also, in that case, it may be a good idea to make the timeout configurable, since the name sounds like requiring the timeout given as a parameter.

File: docs/servo_power_test/src/operate_port.ts [hunk 2]
```diff
@@ -25,14 +25,17 @@ export class OperatePort {
     if (this.port === undefined) return;
     await this.port.open({baudRate: 115200});
   }
-  public async close() {
-    if (this.port === undefined) return;
+  public async readCancel() {
```
Comment (hikalium) [inline docs/servo_power_test/src/operate_port.ts:28]: Maybe it's a good idea to write a function-level documentation to explain the difference between readCancel and close (on top of each function).

Comment (mhiramat) [inline docs/servo_power_test/src/operate_port.ts:28]: Ditto, cancelRead(). but this will invoive many other *Cancel() renaming. So this will be done in another commit.

**Review by mhiramat — COMMENTED** [review]
LGTM, just some cosmetic naming comments.

File: docs/servo_power_test/src/operate_port.ts (hunk 2 above)
Comment (hikalium) [reply docs/servo_power_test/src/operate_port.ts:29]: +1. Thanks for catching this, Masami!
